src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas1/SpaceBookingController.kt

```diff
@@ -140,7 +140,16 @@ class SpaceBookingController(
     bookingId: UUID,
     cas1NewDeparture: Cas1NewDeparture,
   ): ResponseEntity<Unit> {
-    return super.premisesPremisesIdSpaceBookingsBookingIdDeparturePost(premisesId, bookingId, cas1NewDeparture)
+    userAccessService.ensureCurrentUserHasPermission(UserPermission.CAS1_SPACE_BOOKING_RECORD_DEPARTURE)
+
+    extractEntityFromCasResult(
```

Comment:
it may be more readable to use `ensureEntityFromCasResultIsSuccess` instead of `extractEntityFromCasResult` here (and also in the arrival controller above). Technically it does the exact same thing but `ensureEntityFromCasResultIsSuccess` makes it (slightly?) clearer that it will throw an exception if not a success